src/plugins/intel_gpu/tests/test_cases/broadcast_gpu_test.cpp

```diff
@@ -133,45 +139,148 @@ void start_broadcast_test_5d(data_types cldnn_data_type, std::vector<size_t> out
 
 TEST(broadcast_gpu_float, bfyx_1_to_5_w_b_axes_0) {
     std::vector<float> golden_data = {1.0, 1.0, 1.0, 1.0, 1.0};
-    start_broadcast_test<float>(data_types::f32, {5}, {1}, {0}, golden_data);
+    start_broadcast_test<float>(format::bfyx, data_types::f32, {5}, {1}, {0}, golden_data);
+}
+
+TEST(broadcast_gpu_float, b_fs_yx_fsv16_1_to_5_w_b_axes_0) {
+    std::vector<float> golden_data = {1.0, 1.0, 1.0, 1.0, 1.0};
+    start_broadcast_test<float>(format::b_fs_yx_fsv16, data_types::f32, {5}, {1}, {0}, golden_data);
+}
+
+TEST(broadcast_gpu_float, bs_fs_yx_bsv32_fsv16_1_to_5_w_b_axes_0) {
+    std::vector<float> golden_data = {1.0, 1.0, 1.0, 1.0, 1.0};
+    start_broadcast_test<float>(format::bs_fs_yx_bsv32_fsv16, data_types::f32, {5}, {1}, {0}, golden_data);
 }
 
 TEST(broadcast_gpu_uint8_t, bfyx_1_to_5_w_b_axes_0) {
     std::vector<uint8_t> golden_data = {1, 1, 1, 1, 1};
-    start_broadcast_test<uint8_t>(data_types::u8, {5}, {1}, {0}, golden_data);
+    start_broadcast_test<uint8_t>(format::bfyx, data_types::u8, {5}, {1}, {0}, golden_data);
 }
 
 TEST(broadcast_gpu_int64_t, bfyx_1_to_5_w_b_axes_0) {
     std::vector<int64_t> golden_data = {1, 1, 1, 1, 1};
-    start_broadcast_test<int64_t>(data_types::i64, {5}, {1}, {0}, golden_data);
+    start_broadcast_test<int64_t>(format::bfyx, data_types::i64, {5}, {1}, {0}, golden_data);
+}
+
+TEST(broadcast_gpu_int8_t, b_fs_yx_fsv16_1_to_5_w_b_axes_0) {
+    std::vector<int8_t> golden_data = {1, 1, 1, 1, 1};
+    start_broadcast_test<int8_t>(format::b_fs_yx_fsv16, data_types::i8, {5}, {1}, {0}, golden_data);
+}
+
+TEST(broadcast_gpu_int8_t, b_fs_yx_fsv32_1_to_5_w_b_axes_0) {
+    std::vector<int8_t> golden_data = {1, 1, 1, 1, 1};
+    start_broadcast_test<int8_t>(format::b_fs_yx_fsv32, data_types::i8, {5}, {1}, {0}, golden_data);
```

Comment:
This kind of test is not worthy where the feature size < feature block size. 
Could you make the test shapes for blocked layouts for more meaningful ones and compare them with ngraph result?